main.py

```diff
@@ -4257,6 +4257,41 @@ def do_command(self, command, args, sender, channel, tags, isWhisper=False):
                 else:
                     self.message(channel, "Debug mode is off. Debug command disabled.")
                 return
+            if command == "givefreepack" and sender in superadmins:
+                if len(args) < 2:
+                    self.message(channel, "Usage: !givefreepack <username> <booster name> [<amount>] (default 1)", isWhisper)
```

Comment:
Wording: Maybe put the "(default 1)" in the brackets, i.e.:

    Usage: !givefreepack <username> <booster name> [<amount> (default 1)]

This seems clearer to me.